models/ideas.js

```diff
@@ -14,6 +15,19 @@ function createIdea(header, description, askedSum, userId) {
     return newUser;
 }
 
+function updateIdea(ideaId, idea) {
+    let ideas = getIdeas();
+    const idea = idea;
+
+    const updateIdeas = ideas.reduse((updateIdeas, idea) => {
+        if (idea.id === ideaId) {
+            return [...updateIdeas, updateIdea];
```

Comment:
What is `updateIdea` here?

Reply:
it should be idea that we want to update. and it comes in params.